packages/forms/src/UIForm/doc/triggers.md

```diff
@@ -0,0 +1,181 @@
+# Triggers
+
+Triggers are advanced ways to controle properties and errors.
+
+*Warning*
+Use only triggers if you can't perform what you want with another feature listed in the [documentation](../README.md).
+
+## What are triggers ?
+
+Triggers are asynchronous calls that you can perform in your custom widgets.
```

Comment:
You do not need custom widgets to use or understand triggers (do not mix widget dev and form user)

What do you think of
```suggestion
Triggers are custom functions called during event handling. They support asynchronous calls using Promises.
````

Reply:
Changed to avoid custom widgets in the explanation 👍